supervisor/shared/external_flash/external_flash.h

```diff
@@ -3,7 +3,7 @@
  *
  * The MIT License (MIT)
  *
- * Copyright (c) 2017 Scott Shawcroft for Adafruit Industries
+ * Copyright (c) 2013, 2014 Damien P. George
```

Comment:
Damien?

Reply:
Changed. Sometimes I miss them and then git thinks its a move from elsewhere.